test/teos/unit/test_api.py

```diff
@@ -554,3 +555,22 @@ def test_get_appointment_in_responder(internal_api, client, generate_dummy_track
     assert tx_tracker.dispute_txid == r.json.get("appointment").get("dispute_txid")
     assert tx_tracker.penalty_txid == r.json.get("appointment").get("penalty_txid")
     assert tx_tracker.penalty_rawtx == r.json.get("appointment").get("penalty_rawtx")
+
+
+def test_get_subscription_info(internal_api, client, generate_dummy_tracker, monkeypatch):
+    # Need to mock a user
+    internal_api.watcher.gatekeeper.registered_users[user_id].available_slots = 100
+    internal_api.watcher.gatekeeper.registered_users[user_id].subscription_expiry = 700
+    internal_api.watcher.gatekeeper.registered_users[user_id].appointments = {}
+
+    # mock the gatekeeper (user won't be registered if the previous tests weren't ran)
+    monkeypatch.setattr(internal_api.watcher.gatekeeper, "authenticate_user", mock_authenticate_user)
+
+    # Request back the data
+    message = "get subscription info"
+    signature = Cryptographer.sign(message.encode("utf-8"), user_sk)
+    data = {"signature": signature}
+
+    # Next we can request the subscription info
+    r = client.post(get_subscription_info_endpoint, json=data)
+    assert r.status_code == HTTP_OK
```

Comment:
It'll be nice to check that what you get here matches what you set on the registration.